### DISCUSSION TITLE
After screen unlock theme changes to light.

### DISCUSSION THREAD

**Discussion by heejit** [cogentcore/core#1351] [General]
I am using opensuse and kde plasma desktop after screen unlock my applications theme changes to Light Theme.

Theme change on Disconnection of External monitor also.

**Comment by kkoreilly** [comment]
Thank you for reporting this. Are you calling code that overrides the default color theme? If so, can you please paste in the code you are calling related to that and say when you are doing it? That will help me figure out how to fix this. Thank you!

**Reply by heejit** [reply]
once calling following in main function only ->
`colors.SetScheme(true) `

**Reply by kkoreilly** [reply]
Thank you for responding. Please try doing this instead of `colors.SetScheme` *after* calling `core.NewBody`:

```go
core.AppearanceSettings.Theme = core.ThemeDark
```

**Reply by heejit** [reply]
it is not changing the theme to dark instead now it is Light theme on start of program.

**Reply by kkoreilly** [reply]
You also have to add `core.AppearanceSettings.Apply()` after the call above; apologies for forgetting that.

**Reply by heejit** [reply]
it is working now thanks.
